src/linear_algebra/adic_completion.lean

```diff
@@ -210,4 +211,54 @@ h.1.subsingleton
 
 @[priority 100] instance of_subsingleton [subsingleton M] : is_adic_complete I M := {}
 
+open_locale big_operators
+
+lemma le_jacobson_bot [is_adic_complete I R] : I ≤ (⊥ : ideal R).jacobson :=
```

Comment:
Is there no dedicated jacobson radical for a ring? I can imagine that `(⊥ : ideal R).jacobson` becomes ugly and annoying after a while.

Reply:
I don't think we have this. For example `ideal.mem_jacobson_bot` uses `(⊥ : ideal R).jacobson`, and I assume it would have used `jacobson_radical : ideal R` instead if it existed.

Reply:
@adamtopaz would you mind adding that definition to this PR?

Reply:
I will make my Henselian PR dependent on this one.

Reply:
Sure, I'll add it. But I have a lot of teaching stuff to do today, so it may have to wait a little while.

Reply:
@jcommelin I started looking at what it would take to introduce a `jacobson_radical R : ideal R` as the intersection of all maximal ideals of `R`. Now that I'm thinking about it more carefully, I think we should leave the situation as it for now. 
Namely, the nilradical is defined everywhere as `(⊥ : ideal R).radical` and there are several results which rely on the interaction between `ideal.radical` and `ideal.jacobson`. Long story short: to do this refactor properly (if we want to do it at all) should also involve introducing `nilradical R : ideal R`, etc., which will be too big a change to put on top of this PR.